.github/workflows/tests.yml

```diff
@@ -164,7 +164,7 @@ jobs:
 
   centos-8:
     name: CentOS Stream 8
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
Well what do you know, the [CentOS Stream 8 job](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/3250896472/jobs/5335180455) is running and hasn't been canceled. Hooray! :)

Reply:
Just to make sure this wasn't a weird fluke/quirk, I'm going to try and re-run CentOS Stream 8 jobs on the other PRs.

Reply:
Ah, it looks like it was just a momentary fluke. Those jobs are now running, too, despite still using `ubuntu-18.04`.

I'll close the original issue, but I'll keep this PR open, because this is something we'll have to do eventually this year anyway.

Reply:
Re-running Stream 8 jobs on other PRs seems to work at first, but again, the jobs get cancelled midway through (with no logs to show for it).

Reply:
Re-running the Stream 8 jobs a _third_ time, hours later, does in fact lead to passes on all PRs. :facepalm:

Reply:
Just kidding... 4 days later, we're getting Stream 8 jobs cancelled.

I'm almost certain at this point, though, that it's due to the `ubuntu-18.04` brownout, and that this PR fixes the issue.